packages/yoastseo/src/parse/traverse/findAllInTree.js

```diff
@@ -3,10 +3,39 @@
  *
  * @param {Object} tree The tree.
  * @param {function} condition The condition that a node should satisfy to end up in the list.
+ * @param {boolean} recurseFoundNodes=false Whether to recurse into found nodes to see if the condition also applies to sub-nodes of the found node.
+ * If false, as soon as a node is found that satisfies the condition, it is added to the list and no further recursion is done through its children.
+ * If true, the node is added to the list and its children are also checked for the condition.
+ * If they satisfy the condition, they are also added to the list.
+ * This comes with the risk of adding the same node multiple times to the list.
+ *
+ * Example:
```

Comment:
```suggestion
 * @example
```

I think here we can use the JSDoc `@example` as suggested by @mhkuu

Reply:
Thank you for this suggestion. Unfortunately `@exaple` does not really work in this situation where you mix running text with pseudo code.
<img width="698" alt="image" src="https://github.com/Yoast/wordpress-seo/assets/26040749/68ad2a3a-b0d0-42e8-81ec-e3c5bf24a422">